specification/resources/resource-manager/Microsoft.Authorization/stable/2019-09-01/policyAssignments.json

```diff
@@ -0,0 +1,719 @@
+        "x-ms-examples": {
+          "Retrieve a policy assignment by ID": {
+            "$ref": "./examples/getPolicyAssignmentById.json"
+          },
+          "Retrieve a policy assignment with a managed identity by ID": {
+            "$ref": "./examples/getPolicyAssignmentWithIdentityById.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "policyAssignmentId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the policy assignment to get. Use the format '{scope}/providers/Microsoft.Authorization/policyAssignments/{policyAssignmentName}'.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the policy assignment.",
+            "schema": {
+              "$ref": "#/definitions/PolicyAssignment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
+        }
+      },
+      "description": "An error response from a policy operation."
+    },
+    "PolicyAssignmentProperties": {
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the policy assignment."
+        },
+        "policyDefinitionId": {
+          "type": "string",
+          "description": "The ID of the policy definition or policy set definition being assigned."
+        },
+        "scope": {
+          "type": "string",
+          "description": "The scope for the policy assignment."
+        },
+        "notScopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The policy's excluded scopes."
+        },
+        "parameters": {
+          "type": "object",
```

Comment:
I tried the previous api-version before. For the generated SDK, it's hard to know what is the right structure for this field. Is it possible to add the doc link into the description?

Reply:
I've changed parameters to an explicit model.  If this is too much of a breaking change in downstream SDKs I can revert.  However, it will greatly increase quality of life for consumers of the generated code.